Makefile

```diff
@@ -488,22 +478,22 @@ check-go-module:
 ######################################################
 
 .PHONY: release-patch
-release-patch: update-builtin-metadata-release
+release-patch:
 ifeq ($(GITHUB_TOKEN),)
 	@echo "\033[0;31mGITHUB_TOKEN environment variable missing.\033[33m Provide a GitHub Personal Access Token (PAT) with the 'read:org' scope.\033[0m"
 endif
 	@$(DOCKER) run $(DOCKER_FLAGS) \
 		-e GITHUB_TOKEN=$(GITHUB_TOKEN) \
 		-e LAST_VERSION=$(LAST_VERSION) \
 		-v $(PWD):/_src \
-		python:2.7 \
+		cmd.cat/make/git/go/python2/perl \
```

Comment:
Alright I'll admit that I'm not proud of this. But I really kind of dislike having to manage building and tagging and pushing images for these maintenance jobs.

Ideally, we'd be using apko here, I think; but it's macos story is a bit annoying (lima); which we wouldn't have to worry about much if kontain.me just worked on it, but it kind of doesn't...